CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # Changelog
 
+## [2.0.0] **BREAKING CHANGES**
+* Migrate to **null-safety**.
+* Changed named constructors to two separate widgets.
+  * `NumberPicker.integer` is now `NumberPicker`.
+  * `NumberPicker.decimal` is now `DecimalNumberPicker`.
+  * `NumberPicker.horizontal` is now `NumberPicker` with `axis: Axis.horizontal`
+* Removed `infiniteLoop` to make it possible to go to null safety. The feature will come back once the `infitniteListView` package gets null-safe.
+* Removed dialogs containing pickers. I think it's better to just leave it for developers.
+* Removed `animateInt` method and replaced it with just reacting to value changed (see example).
+* Changed animation curve to `Curves.easeOutCubic`.
+* Changed `initialValue` to `value` to avoid confusion from my awful naming.
+* Removed `highlightSelectedValue` as it can be obtained by providing `selectedTextStyle`.
```

Comment:
```suggestion
* Removed `highlightSelectedValue` as it can be obtained by providing `selectedTextStyle`.
* Migrated example to Android embedding v2
```